src/functions/getContent.ts

```diff
@@ -0,0 +1,5 @@
+export const getContent: (code: string) => string = (code: string) => {
```

Comment:
немного лишнего дубля типизации, можно сделать короче
```suggestion
export const getContent = (code: string): string => {
```